CHANGELOG.md

```diff
@@ -55,9 +56,9 @@ All versions prior to 0.9.0 are untracked.
   a `Hashed` parameter to convey the digest used for Rekor entry lookup
   ([#904](https://github.com/sigstore/sigstore-python/pull/904))
 
-* **BREAKING API CHANGE**: `Signer.sign(...)` now takes a `bytes` instead of
-  an `IO[bytes]` for input. Other input types (such as `Hashed` and
-  `Statement`) are unchanged ([#921](https://github.com/sigstore/sigstore-python/pull/921))
+* **BREAKING API CHANGE**: `Signer.sign_artifact(...)` now takes a `bytes`
+  instead of an `IO[bytes]` for input. The `Hashed` parameter option
+  is unchanged ([#921](https://github.com/sigstore/sigstore-python/pull/921))
```

Comment:
I suppose this can just be removed since we already mention that `sign_artifact` has been added above.